WeatherMapDataSource_prometheus.php

```diff
@@ -104,8 +123,8 @@ function ReadData($targetstring, &$map, &$item)
     
             $url = $proto . '://' . $remote_host . ':' . $remote_port . '/api/v1/query?query=';
     
-            $in_query  = 'irate(' . $in_series  . '{ifName="' . $intf . '",instance="' . $instance . '"}[2m])*8';
-            $out_query = 'irate(' . $out_series . '{ifName="' . $intf . '",instance="' . $instance . '"}[2m])*8';
+            $in_query  = 'irate(' . $in_series  . '{ifName="' . $intf . '",instance="' . $instance . '"}[10m])*8';
```

Comment:
Non-blocking:
Funny, I've moved to 20m on some devices that are slow as hell to be polled ( nexus7k + lots of fexes ) and was looking at making this tunable in TARGET ( with a default to some value )

Reply:
These changes were left over from initial lab testing where we currently only poll every 5m. I've reverted them since is was unintended churn.